addons/mail/static/src/core/common/suggestion_hook.js

```diff
@@ -86,6 +90,12 @@ export class UseSuggestion {
         let text = "";
         if (this.comp.composerService.htmlEnabled) {
             const selection = this.comp.editor.shared.selection.getEditableSelection();
+            if (
+                !isTextNode(selection.startContainer) ||
+                !isContentEditable(selection.startContainer)
+            ) {
+                return;
+            }
```

Comment:
What cases do we want to exclude with these 2 conditions?

Reply:
We're actually missing a case here, with the editor, if `startContainer` is different than `endContainer` we should also abort.
Same as `if (start !== end) {`, but we didn't need to worry about container when we just had text.

Reply:
This might be easier checked with `isCollapsed` with the editor rather than making the 2 conditions separately?